src/stores/analytics.ts

```diff
@@ -81,7 +83,15 @@ export const useAnalytics = create<State & Actions>()((set, get) => {
       const { amp, userId, deviceId } = get()
 
       const { parentOrigin } = useParentData.getState()
-      const commonProperties = { from: parentOrigin }
+
+      const slug = Router.query.slug as string
+      const chatId = slug && getIdFromSlug(slug)
+      const commonProperties = {
+        from: parentOrigin,
+        pathname: Router.asPath,
+        query: { ...Router.query, chatId },
```

Comment:
Amplitude cant work w/ objects, so lets use just hubId, chatId, good enough for now

Reply:
updated